docs/build/sdk/indexs.md

```diff
@@ -33,106 +33,69 @@ the minimum amount.
-estimation otherwise "false" |
+**Object Arguments** 
+| Name                  | Type                   | Description                                                    |
+| --------------------- | ---------------------- | -------------------------------------------------------------- |
+| signer                | KeyringPairOrExtSigner | The actual signer provider to sign the transaction             |
+| oracle                | string                 | The address that will be responsible for reporting the market  |
+| period                | MarketPeriod           | Start and end block numbers or unix timestamp of the market    |
+| marketType            | MarketTypeOf           | `Categorical` or `Scalar`                                      |
+| mdm                   | MarketDisputeMechanism | Dispute settlement can be authorized, court or simple_disputes |
+| metadata              | DecodedMarketMetadata  | A hash pointer to the metadata of the market                   |
+| amount                | string                 | The amount of each token to add to the pool                    |
+| weights               | string[]               | List of relative denormalized weights of each asset            |
+| callbackOrPaymentInfo | boolean                | `true` to get txn fee estimation otherwise `false`             |
 
 [Code snippet](https://github.com/Whisker17/sdk-demo/tree/main/src/index/createCpmmMarketAndDeployAssets.ts)
 
-## createCategoricalMarket
+## createMarket
 
-You can use this function to create a categorical market in the Zeitgeiest.
+You can use this function to create categorical or scalar market using below arguments.
 
 ```typescript
 const sdk = await SDK.initialize(endpoint);
 
-const marketId = await sdk.models.createCategoricalMarket(
+const marketId = await sdk.models.createMarket({
   signer,
   oracle,
-  marketPeriod,
-  advised,
-  mdm,
-  cpmm,
+  period: marketPeriod,
   metadata,
-  false
-);
+  creationType: advised ? `Advised` : `Permissionless`,
+  marketType: { Scalar: bounds ? bounds : [0, 100] },
+  mdm,
+  scoringRule: cpmm ? `CPMM` : `RikiddoSigmoidFeeMarketEma`,
+  callbackOrPaymentInfo: false,
+});
 ```
 
-**Arguments** | Name | Type | Introduction | | ---- | ---- | ------------ | |
-signer | KeyringPairOrExtSigner | The actual signer provider to sign the
-transaction. | | oracle | string |The address that will be responsible for
-reporting the market. | | period | MarketPeriod |Start and end block numbers or
-unix timestamp of the market. | | creationType | string |"Permissionless" or
-"Advised", Advised as default | | mdm | MarketDisputeMechanism |Dispute
-settlement can be authorized, court or simple_disputes | | scoringRule | string
-| scoringRule you choose, CPMM as default| | metadata | DecodedMarketMetadata
-|Market metadata | | callbackOrPaymentInfo | |"true" to get txn fee estimation
-otherwise "false" |
+**Object Arguments** 
```

Comment:
```suggestion
**Object Arguments**

```

Otherwise, the table doesn't render correctly.

Reply:
There's some other issue which is causing table rendering. @Whisker17 is working on it, I think.